### ISSUE TITLE
transition: AccessibilityMap: remove water

### ISSUE THREAD

**Issue by kaligrafy** [chairemobilite/transition#116]
remove water polygons from accessibility polygons
ideas: try accessing mapbox background layer or use a web service to import water from osm (should we remove real time after each calculation or should we force the user to first import water polygons and use this import for accessibility maps)

**Comment by kaligrafy** [comment]
Water is not included in OSRM and don't need since there are no paths in water. We just need to substract water from the goejson polygons. But we could also change the isochrone method to use valhalla, which could only use a small buffer around paths, so we don't need to remove water.

**Comment by GabrielBruno24** [comment]
I see, I thought the generation of the polygons was handled trRouting side. Still, we can't just remove the water regions from the map and call it a day, we'd have to do something at each step. For example, if we get a map that goes over a river and back onto land, and then delete the water, we'd get a nonsensical sliver of accessibility that's not connected to anything. See:

![Image](https://github.com/user-attachments/assets/7dfcb7d6-df6d-4dec-b5d2-648ae918d6c8)

**Comment by kaligrafy** [comment]
True, but this would be a first step. The best is to use real network isochrones like in valhalla: https://valhalla.github.io/valhalla/api/isochrone/api-reference/
